lte/gateway/c/session_manager/SessionState.cpp

```diff
@@ -804,10 +813,20 @@ SessionConfig SessionState::get_config() const {
   return config_;
 }
 
+std::string SessionState::get_subscriber_id() const {
```

Comment:
We already have `get_imsi` that returns the same value, so we can get rid of this.

Reply:
Removed the function get_subscriber_id() and replaced with get_imsi()